recipes/openconnect-sso/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openconnect-sso" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vlaci/openconnect-sso/releases/download/v{{ version }}/openconnect-sso-{{ version }}.tar.gz
+  sha256: a7bde0a8dd35d7bab3c66f28b420c343c270a7ee45e98e04bfa6ecc6958f5c5b
+
+build:
+  skip: True  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
```

Comment:
I think you should specify entry points

Reply:
can this be made into a no arch package?

Reply:
> can this be made into a no arch package?

No, because we don't have Windows build for `openconnect` (and perhaps a few other dependencies? I didn't check) yet. See the linter error https://github.com/conda-forge/staged-recipes/pull/19359#discussion_r905698343.

Reply:
> I think you should specify entry points

Done in 58d128d.